### ISSUE TITLE
Which is the best way to test the deployment site?

### ISSUE THREAD

**Issue by green0701** [tntim96/JSCover#303]
I deploy my test site to IIS (my js need deploy). So i use like 192.168.8.111:80/index.html to test my js. (I use Qunit) So which is the best way to use JSCover collect report?

And when I execute "java -jar JSCover-all.jar -fs ./test ./testJSCover"
Some error give me.
[test.zip](https://github.com/tntim96/JSCover/files/6883131/test.zip)

![image](https://user-images.githubusercontent.com/83161005/127105394-1afdda7f-a0b5-46b0-9fbf-ef65fcfd15d1.png)

**Comment by tntim96** [comment]
> Which is the best way to test the deployment site?

See this example: https://github.com/tntim96/JSCover/tree/master/examples/localStorage-file-system

I'll have to investigate what specifically is causing that error.

**Comment by AngelaWanChina2021** [comment]
please let me know how to fix the issue I meet the same issues

**Comment by tntim96** [comment]
@AngelaWanChina2021 or @green0701 can you isolate the bit of JS code that causes this error?

**Comment by angelikaYang** [comment]
I have the same issue, any update here? how to fix this? I start the jscover with fs

**Comment by tntim96** [comment]
I've released it in the 2.0.15 SNAPSHOT if you'd like to test.

```XML
<repositories>
    <repository>
        <id>sonatype-nexus-snapshots</id>
        <name>Sonatype Nexus Snapshots</name>
        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Or build the latest JAR yourself or download from [here](http://jscover.sourceforge.net/latest/JSCover-all.jar).

Let me know if that fixes your issue and I'll build a release.

**Comment by green0701** [comment]
It looks great.

**Comment by tntim96** [comment]
Fixed in release v2.0.15
